docs/src/_includes/partials/docs-footer.html

```diff
@@ -4,7 +4,7 @@
             {% include 'components/social-icons.html' %}
         </div>
         <div class="docs-copyright">
-            &copy; OpenJS Foundation and other contributors, <a href="https://www.openjsf.org">www.openjsf.org</a>
+            <p>{{ site.footer.copyright | safe }}</p>
```

Comment:
This `<p>` seems to add a bit of vertical space between the copyright line and the theme switcher. Is it intentional?

This PR's preview:

![image](https://user-images.githubusercontent.com/44349756/173875842-8a6397bb-c394-4e82-9a7c-58931411065d.png)

Current head version (https://new.eslint.org/docs/head/):

![image](https://user-images.githubusercontent.com/44349756/173876161-cd61897f-c1af-4cf4-86ce-e0de1174a5e1.png)

Reply:
```suggestion
            {{ site.footer.copyright | safe }}
```